admin/src/main/java/org/bahmni/module/admin/observation/handler/Form1CSVObsHandler.java

```diff
@@ -46,4 +46,9 @@ public Form1CSVObsHandler(CSVObservationHelper csvObservationHelper) {
         return observations;
     }
 
+    @Override
+    public List<EncounterTransaction.Observation> handle(EncounterRow encounterRow, boolean shouldPerformForm2Validations) throws ParseException {
+        return null;
```

Comment:
I am skeptical of "null" being returned. I am guessing there should be no case when a caller called this method on Form1CSVObsHandler. If so, then maybe it should throw a ParseException with message saying, called with form2validation on a Form1 handler. That will be better, than some chain ignoring this error, and instead ending up printing null / throwing a null pointer exception down the line. What do u think?  And, then u can add a test to say, if someone calls this method on Form1  handler, it should throw a ParseException.

Reply:
Thanks Gurpreet for reviewing this. This is a dummy method for form1Obs handler and will not executed from existing end point. This is called from ObservationMapper which contains list of Obs handlers. Returning null from this method and gracefully handling in the called method [ObservationMapper.getObservations(encounterRow,shouldPerformForm2Validations)]